### DISCUSSION TITLE
Unraid - XML Config with TOTP / 2FA Working

### DISCUSSION THREAD

**Discussion by NightCrawler1016** [mrlt8/docker-wyze-bridge#673] [Show and tell]
Hello all, I was running into a wall trying to get this working on Unraid and finally figured it out. Posting here because I was unable to find any answers on google and I hope this helps someone =)

I'm not sure how to add an xml file here so if you use the attachment, make sure to change it from .txt to .xml for unraid.

[docker-wyze-bridge.txt](https://github.com/mrlt8/docker-wyze-bridge/files/10428639/docker-wyze-bridge.txt)

### Prerequisites

- Dynamix File Manager

- TOTP / 2FA Reset from Wyze App
  1. If you don't know your wyze TOTP / 2FA token, you will need to `disable` TOTP / 2FA and `then re-enable` from within the wyze application on iOS / Android

-  Import of XML file to Unraid

**How to import docker container .xml to unraid as of 6.11.5**
  1. From the Main tab click on the `button under View` to the right of your boot flash drive
  3. Navigate to `config->plugins->dockerMan->templates-user`
  4. Click the `Upload` button at the bottom right of the screen
  5. Choose your .xml file and click `Done`

### After you have your TOTP / 2FA Token

1. Download the above `docker-wyze-bridge.txt` file
2. Rename `docker-wyze-bridge.txt` to `docker-wyze-bridge.xml`
3. Import the .xml to unraid
4. Navigate to `Docker` and click `Add Container`
6. Chose `docker-wyze-bridge` from the template list
7. Add your wyze account email address
8. Add your wyze account password (the one you use in the mobile app)
9. Add the TOTP / 2FA token from the wyse app earlier in `TokenFromWyzeApp`
10. Click `Apply`

That's it! The only changes I made from the original docker-wyze-bridge from the unraid community apps by mrlt8/wyze-bridge are as follows:

- Added WebUI - http://[IP]:[PORT:5000]/
- Added Host Port 3 - TCP 5000
- Added TOTP - TOTP_KEY
- Added Enable Audio: - ENABLE_AUDIO

EDIT: Forgot to add support for audio...

**Comment by coldmantis** [comment]
I'm new to unraid and I been trying to get this to work for a few days now. Google also didn't help with getting the app from unraid to load the web ui so I decided to follow your instructions, however I'm stuck on this part "Click the Upload button at the bottom right of the screen" I have no upload button and I did install Dynamix System Buttons before hand. Any ideas?

https://prnt.sc/8gVPl6JXdGRn

**Reply by NightCrawler1016** [reply]
Hey @coldmantis try adding the Dynamix File Manager plugin and let me know if that gives you the correct buttons.

**Reply by coldmantis** [reply]
Thanks I figured it out before you replied. Thanks for the instructions. My Wyze dock finally works.
